components/DoveadmComponent.vue

```diff
@@ -18,115 +18,129 @@ const d = Object.fromEntries(Object.entries(data.doveadm).filter(([k, v]) =>
-   <tbody>
-    <tr v-if="v.man_link">
-     <th style="text-align: right;">Man Page</th>
-     <td v-html="v.man_link" />
-    </tr>
+   <table v-if="v.man_link || v.fields || v.added || v.changed || v.deprecated || v.removed">
+    <tbody>
+     <tr v-if="v.man_link">
+      <th style="text-align: right;">Man Page</th>
+      <td v-html="v.man_link" />
+     </tr>
 
-    <tr v-if="v.added || v.changed || v.deprecated || v.removed">
-     <th style="text-align: right;">Changes</th>
-     <td>
-      <ul>
-       <li v-if="v.added" v-for="elem in v.added">
-        <span v-html="elem.version" />
-        <span v-html="elem.text" />
-       </li>
-       <li v-if="v.changed" v-for="elem in v.changed">
-        <span v-html="elem.version" />
-        <span v-html="elem.text" />
-       </li>
-       <li v-if="v.deprecated" v-for="elem in v.deprecated">
-        <span v-html="elem.version" />
-        <span v-html="elem.text" />
-       </li>
-       <li v-if="v.removed" v-for="elem in v.removed">
-        <span v-html="elem.version" />
-        <span v-html="elem.text" />
-       </li>
-      </ul>
-     </td>
+     <tr v-if="v.added || v.changed || v.deprecated || v.removed">
+      <th style="text-align: right;">Changes</th>
+      <td>
+       <ul>
+        <li v-if="v.added" v-for="elem in v.added">
+         <span v-html="elem.version" />
+         <span v-html="elem.text" />
+        </li>
+        <li v-if="v.changed" v-for="elem in v.changed">
+         <span v-html="elem.version" />
+         <span v-html="elem.text" />
+        </li>
+        <li v-if="v.deprecated" v-for="elem in v.deprecated">
+         <span v-html="elem.version" />
+         <span v-html="elem.text" />
+        </li>
+        <li v-if="v.removed" v-for="elem in v.removed">
+         <span v-html="elem.version" />
+         <span v-html="elem.text" />
+        </li>
+       </ul>
+      </td>
     </tr>
 
-    <tr v-if="v.fields">
-     <th style="text-align: right;">Return Values</th>
-     <td>
-      <table>
-       <thead>
-        <tr>
-         <th>Key</th>
-         <th>Value</th>
-        </tr>
-       </thead>
-       <tbody>
-        <tr v-for="elem in v.fields">
-         <td><code>{{ elem.key }}</code></td>
-         <td v-html="elem.value" />
-        </tr>
-       </tbody>
-      </table>
-     </td>
-    </tr>
-   </tbody>
-  </table>
+     <tr v-if="v.fields">
+      <th style="text-align: right;">Return Values</th>
+      <td>
+       <table>
+        <thead>
+         <tr>
+          <th>Key</th>
+          <th>Value</th>
+         </tr>
+        </thead>
+        <tbody>
+         <tr v-for="elem in v.fields">
+          <td><code>{{ elem.key }}</code></td>
+          <td v-html="elem.value" />
+         </tr>
+        </tbody>
+       </table>
+      </td>
+     </tr>
+    </tbody>
+   </table>
 
-  <span v-if="v.text" v-html="v.text"></span>
+   <span v-if="v.text" v-html="v.text"></span>
```

Comment:
Similar to my comment in #1000 this block will contain `<p>` in practice and `<span>` must not contain those. Change to `<div>`.